swan-lake/learn/documenting-ballerina-code.md

```diff
@@ -25,7 +25,7 @@ Ballerina documentation design and usage is aligned with the package and module
 
 ## Writing Ballerina Documentation
 
-Ballerina Flavored Markdown documentation is a first class syntax in the Ballerina language. The `#` at the beginning of a line denotes a line of documentation. If necessary, you can have multiple lines of documentation, which you can group together.
+Ballerina Flavored Markdown documentation is a first-class syntax in the Ballerina language. The `#` at the beginning of a line denotes a line of documentation. If necessary, you can have multiple lines of documentation, which you can group together.
```

Comment:
```suggestion
Ballerina Flavored Markdown documentation is a first-class syntax in the Ballerina language. The `#` at the beginning of a line denotes a line of documentation. If necessary, you can have multiple lines of documentation, which you can be grouped together.
```

Reply:
No need to have the passive voice form here.